app/theme/client/imports/forms/input.css

```diff
@@ -55,7 +55,7 @@ textarea.rc-input__element {
 	&__element {
 		width: 100%;
 		height: 2.5rem;
-		padding: 0 1rem;
+		padding: 0.5rem 0 0.5rem 1rem;
```

Comment:
I'm not sure if it will not trigger undesired side effects on all inputs. Maybe target the CSS rule by `type` attribute (e.g. `&__element[type=time]`) and use the pseudoselectors `::-webkit-inner-spin-button` and `::-webkit-outer-spin-button` since it's probably limited to some browsers.